### DISCUSSION TITLE
Is there any other OS?

### DISCUSSION THREAD

**Discussion by brambora69123** [fnuecke/oc2#262] [General]
I was wondering if there is any other linux distro I could use (like ubuntu) is it even possible?

**Comment by spike11302000** [comment]
most likely not as these computers only have 32mb of ram. you might be able to get arch or void working on it but there wont be much memory left for user apps. another option is to use a something like netbsd when riscv support is available

**Reply by atirutw** [reply]
You'd also need the ability to flash custom firmwares since the default firmware is Linux.

**Reply by oof2472643** [reply]
the archlinux reqiurements are 512mb of ram and 1gb of storage

**Reply by atirutw** [reply]
I don't think you can even run SystemD considering how little memory you get to have.
